test/integration/import_test.py

```diff
@@ -0,0 +1,106 @@
+import csv
+import pytest
+import pyexasol
+from inspect import cleandoc
+
+
+@pytest.fixture
+def connection(dsn, user, password, schema):
+    with pyexasol.connect(
+        dsn=dsn, user=user, password=password, schema=schema, compression=True
+    ) as con:
+        yield con
+
+
+@pytest.fixture
+def table_name():
+    yield "CLIENT_NAMES"
+
+
+@pytest.fixture
+def empty_table(connection, table_name):
+    ddl = cleandoc(f"""
+    CREATE TABLE IF NOT EXISTS {table_name}
+    (
+        FIRSTNAME       VARCHAR(255),
+        LASTNAME        VARCHAR(255)
+    );
+    """)
+    connection.execute(ddl)
+    connection.commit()
+
+    yield table_name
+
+    ddl = f"DROP TABLE IF EXISTS {table_name};"
+    connection.execute(ddl)
+    connection.commit()
+
+
+@pytest.fixture
+def data(faker):
+    yield [(faker.first_name(), faker.last_name()) for _ in range(0, 10)]
+
+
+@pytest.fixture
+def swaped_data(data):
+    yield [(lastname, firstname) for firstname, lastname in data]
+
+
+@pytest.fixture
+def csv_file(tmp_path, data):
+    file = tmp_path / "names.csv"
+    with open(file, "w", newline="") as csvfile:
+        writer = csv.writer(csvfile, dialect="unix")
+        for row in data:
+            writer.writerow(row)
+    yield file
+
+
+@pytest.mark.etl
+def test_import_with_custom_csv_format(connection, empty_table, csv_file, data):
```

Comment:
How is this a custom csv format, it seems to unix dialect , is this something special?

Reply:
No maybe the name is not ideal, I don't recall why I'll rename it.